crates/db_schema/src/source/password_reset_request.rs

```diff
@@ -10,11 +10,13 @@ pub struct PasswordResetRequest {
   pub token_encrypted: String,
   pub published: chrono::NaiveDateTime,
   pub local_user_id: LocalUserId,
+  pub expires_at: chrono::NaiveDateTime,
```

Comment:
I dont see any reason to add a new column here. Would be enough to delete the row when the password reset was performed successfully.

Reply:
Note that this would enable malicious actors to bypass the daily limit I added in #3344

Reply:
What @sunaurus said, I don't see any real harm in adding the extra column. If expiration duration ever becomes configurable it might be nice to be able to tell when the actual expiration time was, vs having to figure out what the expiration duration was configured to at the time and add it to the published timestamp.

Reply:
Makes sense about the daily limit. But for that a boolean column `expired` would be enough, I dont see any reason for a second timestamp.

Reply:
I could change it to bool if you like, I just figured storing when it expired might be useful information in some case.

Reply:
But expiration would always be 24h after publish time, no? So all this needs is a bool `used` or similar.

Reply:
Yeah, I suppose a bool would work fine, I've just always seen it implemented with a timestamp so that's what I went with :shrug: I'll change it later when I have time.

Reply:
We should also add a daily job to delete these based on their published time > than a day, otherwise the password reset request table will grow pointlessly large.

Reply:
Why would the column be marked as `used` before its been used? It should be set to true after its been used.

Reply:
It would be marked `used` if a new token is generated before 24 hours. Maybe it's not too big of a deal and I'm overthinking it, I'll get it updated this weekend.